server/proxy/proxy.go

```diff
@@ -312,7 +289,7 @@ func (p *Proxy) notifyJoinLeave(cc *apps.Context, subject, botSubject apps.Subje
 }
 
 func (p *Proxy) GetStatic(appID apps.AppID, path string) (io.ReadCloser, int, error) {
-	m, err := p.store.Manifest.Get(appID)
+	app, err := p.store.App.Get(appID)
```

Comment:
In the install dialog the app icon is used despite the app not being installed (yet) I'm not sure if on of your future PR will change that, but for now this change will break the icon in the dialog.

Reply:
Ack, will check, might still need to fix, I went back/forth on this one a few times now.